src/menu/ExportIEDParams.ts

```diff
@@ -0,0 +1,303 @@
+            return `DAI[name="${path}"]`;
+          }
+          // Every element(s) between the DOI and DAI element is always a SDI element.
+          return `SDI[name="${path}"]`;
+        })
+        .join(' > ');
+
+      const daiValueElement = lnElement.querySelector(daiSelector + ' Val');
+      return daiValueElement?.textContent ?? null;
+    }
+    return null;
+  }
+
+  /**
+   * First check if there is an instance element found (DAI) found, otherwise search in the Template section.
+   *
+   * @param lnElement         - The LN Element used as starting point for the search.
+   * @param dataAttributePath - The names of the DO(I)/SD(I)/DA(I) Elements to search for.
+   */
+  private getDataAttributeValue(
+    lnElement: Element,
+    dataAttributePath: string[]
+  ): string {
+    let value = this.getDataAttributeInstanceValue(
+      lnElement,
+      dataAttributePath
+    );
+    if (!value) {
+      value = this.getDataAttributeTemplateValue(lnElement, dataAttributePath);
+    }
+    return value ?? '';
+  }
+
+  /**
+   * Retrieve the list of elements found by the selector or if no selector defined the IED element.
+   *
+   * @param iedElement       - The IED element that will be used to search below if useOwnerDocument is false.
+   * @param selector         - If passed the CSS selector to search for the elements.
+   * @param useOwnerDocument - If false will use the IED element to search below, otherwise the full document.
+   */
+  private getElements(
+    iedElement: Element,
+    selector: string | undefined,
+    useOwnerDocument: boolean
+  ): Element[] {
+    let elements: Element[] = [iedElement];
+    if (selector) {
+      const iedName = iedElement.getAttribute('name') ?? '';
+      const substitutedSelector = this.getSelector(selector, iedName);
+      if (useOwnerDocument) {
+        elements = Array.from(
+          iedElement.ownerDocument.querySelectorAll(substitutedSelector)
+        );
+      } else {
+        elements = Array.from(iedElement.querySelectorAll(substitutedSelector));
+      }
+    }
+    return elements;
+  }
+
+  /**
+   * Create a single line of values for the CSV File.
+   *
+   * @param configuration - The configuration with values to retrieve.
+   * @param iedElement    - The IED Element for which to retrieve the values.
+   */
+  private cvsLine(configuration: Configuration, iedElement: Element): string[] {
+    return configuration.columns.map(column => {
+      const elements = this.getElements(
+        iedElement,
+        column.selector,
+        column.useOwnerDocument ?? false
+      );
+
+      return elements
+        .map(element => {
+          if (column.dataAttributePath) {
+            return this.getDataAttributeValue(
+              element,
+              column.dataAttributePath
+            );
+          }
+          return this.getValue(element, column.attributeName);
+        })
+        .filter(value => value!)
+        .join(' / ');
```

Comment:
Why the `/` here. Is it something the CSV library needs?

Reply:
If multiple matches are found the values are just joined, separated by a '/'.
This means multiple matches for the same selector.

Reply:
Git it! Thx